tests/integration_tests/performance/test_block_performance.py

```diff
@@ -20,12 +20,13 @@
 from framework.utils import (
     get_cpu_percent,
     get_kernel_version,
+    get_instance_type,
     is_io_uring_supported,
     CmdBuilder,
     DictQuery,
     run_cmd,
 )
-from framework.utils_cpuid import get_cpu_model_name, get_instance_type
+from framework.utils_cpuid import get_cpu_model_name
```

Comment:
How about moving `get_cpu_model_name()` to `utils` along with `get_instance_type()`? It is also used out of context of CPUID-related tests.

Reply:
Long-term I think we should have some "properties" fixture where we can pull common test environment information like this, rather than each test having to request it on its own. The metrics PRs will bring some of that.